src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Resident.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import software.amazon.awssdk.annotations.NotNull
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "resident")
+data class Resident(
+  @Id
+  @NotNull
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long = -1,
+
+  val forename: String,
+
+  val surname: String,
+
+  val phoneNumber: String,
+
+  val relation: String,
+
+  val dateOfBirth: LocalDate,
+
+  val age: Int,
```

Comment:
think this is null for main resident? presumably as they assume they will be over 18

Reply:
fixed